unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Texture/AssetPromise_EmptyTexture.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using DCL.Helpers;
+using MainScripts.DCL.Analytics.PerformanceAnalytics;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.Video;
+
+namespace DCL
+{
+    public class AssetPromise_EmptyTexture : AssetPromise_Texture
```

Comment:
We can use `Texture2D.whiteTexture` and avoid this class.

Reply:
Texture2D.whiteTexture is read-only texture. I tried to use it. But it has problems with the VideoTexture re-assignment.

And this class is to have a Promise of an EmptyTexture to mantain the same system to assign the asset. And without modifying core parts of our system. If you suggest something simplier, it should help a lot 🙏🙌

Reply:
Oh ofc, I didnt think about the texture needed to be writeable (which is obvious now that I think about it), it's fine, let's keep it that way.

Reply:
I didnt think about the texture readabiliity...

Yeah, let's keep it as it is. This is the problem with forcing ECS in an environment not suited for it, it's so inflexible we will never find a clean solution for this.

Ideally we could use the dependency of inversion principle and have a resolver gathering the texture but of course that's impossible in the current architecture with the material being a component itself.
Anyway, let's go with this!